### PR TITLE
FarmBeats Base Client

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: eng/.docsettings.yml [hunk 1]
```diff
@@ -180,6 +180,7 @@ known_content_issues:
   - ['sdk/textanalytics/azure-ai-textanalytics/swagger/README.md', '#3113']
   - ['sdk/translation/azure-ai-documenttranslator/swagger/README.md', '#3113']
   - ['sdk/webpubsub/azure-messaging-webpubsub/swagger/README.md', '#3113']
+  - ['sdk/farmbeats/azure-verticals-agrifood-farming/swagger/README.md', '#3113']
```
Comment (srnagar) [inline eng/.docsettings.yml:183]: nit: fix order

**Review by srnagar — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
@jianghaolu The swagger for this file has been updated. See https://github.com/Azure/azure-rest-api-specs/pull/14408.

Please create a PR with re-generated package using the latest swagger

cc @lmazuel

**Pull request merged**